Web_Development/Javascript/7.The DOM/DOM - Examples/README.md

```diff
@@ -0,0 +1,10 @@
+# The DOM Javascript Examples
+
+## 1. Background Color Changer
+## 2. Counter
+## 3. Tab Switching
+## 4. Modal
+## 5. Sort courses
+<br>
+Contributor: <a href="https://github.com/amankumar11">Aman Kumar</a>
+
```

Comment:
No need of this.

Reply:
Required changes made in the latest commit.